Queue.hpp

```diff
@@ -45,8 +45,10 @@ class Queue
      * Call will block until response is given with respondTo().
      *
      * @param msg Request message. Is put to the queue so it can be retrieved from it with get().
+     * @param timeoutMillis How many ms to wait for response until timeout happens.
+     *                      0 = wait indefinitely.
```

Comment:
Documentation should mention that nullptr is returned in case of timeout

Reply:
It is the purpose of this modification, isn't it?